src/controller/CHIPDeviceController.h

```diff
@@ -831,7 +832,10 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
     Callback::Callback<OnNOCChainGeneration> mDeviceNOCChainCallback;
     SetUpCodePairer mSetUpCodePairer;
     AutoCommissioner mAutoCommissioner;
-    CommissioningDelegate * mCommissioningDelegate = nullptr;
+    CommissioningDelegate * mDefaultCommissioner =
+        &mAutoCommissioner; // Commissioning delegate to call when PairDevice / Commission functions are used
```

Comment:
This `= &mAutoCommissioner` overrides the logic in `Init`. It should defualt to null.

Reply:
see next commit.